braintree/search.py

```diff
@@ -1,5 +1,12 @@
 class Search:
+	"""
+		Collection of classes used to build search queries.
+		
+		Each Builder class defines one or more methods that returns a Node
+		object with the name of the field, the comparator, and the value.
+	"""
 	class IsNodeBuilder(object):
+		""" Builds a query for value equality. """
```

Comment:
Could you remove the space padding on either side of this string? Eg, it should look like this:

```py
"""Builds a query for value equality."""
```